src/scheduler/limits.c

```diff
@@ -2913,6 +2913,18 @@ clear_limres(void)
 	free_resource_list(limres);
 	limres = NULL;
 }
+/**
+ *      @brief returns a linked list of resources being limited.
+ *
+ *      @par This is to be treated as read-only.  Modifying this will adversely
+ *           affect the limits code
+ *
+ *      @return schd_resource *
+ */
+schd_resource *query_limres()
```

Comment:
The type and function name should be on separate lines. If it takes no arguments, please add "void" in parens.